frontend/src/components/common/user-avatar/user-avatar.tsx

```diff
@@ -65,7 +69,12 @@ export const UserAvatar: React.FC<UserAvatarProps> = ({
         width={imageSize}
       />
       <ShowIf condition={showName}>
-        <span className={`ms-2 me-1 ${styles['user-line-name']}`}>{displayName}</span>
+        <span
+          className={concatCssClasses(styles['user-line-name'], 'ms-2', 'me-1', {
+            'fst-italic': isGuest
```

Comment:
Do we really want this in italics? I know we talked about this and I thought the consenus was no, we don't?

Reply:
hm. now that I see it... It looks a bit weird. The guests are already clearly indicated. I second this.